server/utils/sendBill.js

```diff
@@ -0,0 +1,71 @@
+const nodemailer = require('nodemailer');
+
+const sendTheBill = async (
+  to,
+  subject,
+  { total, hourPrice, hourNumber, description, client, provider },
+  next
+) => {
+  try {
+    const smtpTransport = nodemailer.createTransport({
+      service: 'Gmail',
+      auth: {
+        user: process.env.NODEMAILER_SENDER_EMAIL,
+        pass: process.env.SENDER_EMAIL_PASSWORD,
+      },
+    });
+
+    const msg = {
+      from: process.env.NODEMAILER_SENDER_EMAIL,
+      to,
+      subject,
+      html: `
+      <h2>Service Seeker</h2>
+      <div style="padding: 10px; background-color: #25D595; color: #fff"><span>INVOICE TO: </span>${client}</div>
+      <div style="padding: 10px; background-color: #25D595; color: #fff"><span>Date: </span>${new Date().toDateString()}</div>
+      <hr/>
+      <table style="border:none;border-collapse:collapse;border-color:#aaa;border-spacing:0;">
+      <thead>
+        <tr>
+          <th style="text-align:left;vertical-align:top; background-color:#f38630;border-color:#aaa;border-style:solid;border-width:0px;color:#fff;
+          font-family:Arial, sans-serif;font-size:14px;font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;"><span style="font-weight:bold;">Client</span></th>
+          <th style="text-align:left;vertical-align:top; background-color:#f38630;border-color:#aaa;border-style:solid;border-width:0px;color:#fff;
+          font-family:Arial, sans-serif;font-size:14px;font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;"><span style="font-weight:bold">Provider</span></th>
```

Comment:
refactor the html content, remove all the duplicate style
* if you need to use colors inside the email, please use our site colors